bin/interactive-install.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
+
+set -eu
+shopt -s extglob
+
+echo -e "\e[32mWelcome to the secureCodeBox!\e[0m"
+echo -e "This interactive installation script will guide you through all the relevant installation steps in order to have you ready to scan."
+
+echo -e "Start ? [y/n]"
```

Comment:
It would be great if you add some additional CLI flags in the case i want to install everything without the interactive dialog, eg `--all` with our defaults (for namespaces). This would help for example to use this script in an vagrant context where we could provide a fully all-in-one vagrant box with autoinstalling everything (minikube + SCBv2)